maykin_2fa/views.py

```diff
@@ -146,6 +147,11 @@ def get_context_data(self, form, **kwargs):
         context["title"] = _("Use recovery token")
         return context
 
+    def get_form_list(self):
+        form_list = super().get_form_list()
+        form_list[self.BACKUP_STEP] = CustomBackupTokenForm
+        return form_list
```

Comment:
Same remark as on the form, we can rely on just the template changes to show the errors
```suggestion
```